rw-stream-sink/README.md

```diff
@@ -0,0 +1,7 @@
+This crate provides the `RwStreamSink` type. It wraps around a `Stream + Sink` that produces
+byte arrays, and implements `AsyncRead` and `AsyncWrite`.
+
+Each call to `write()` will send one packet on the sink. Calls to `read()` will read from
+incoming packets.
+
+> **Note**: This crate is mostly a utility crate and is not at all specific to libp2p.
```

Comment:
Maybe make this "Although this crate is hosted in the `libp2p` repo, it is purely a utility crate and not at all specific to libp2p", so that users that see it on crates.io or docs.rs or whatever don't get confused

Reply:
We're not sure that this crate is going to remain in the `libp2p` repo, but if we move it out then this note will need to be removed anyway, so I'll make the change.